test/e2e/backup_restore_test.go

```diff
@@ -197,13 +197,13 @@ func assertNumEntries(cacheName, pod string, numEntries int, client ispnclient.H
 	}
 
 	/*
-	body, err := ioutil.ReadAll(rsp.Body)
-	tutils.ExpectNoError(err)
-	numRead, err := strconv.ParseInt(string(body), 10, 64)
-	tutils.ExpectNoError(err)
-	if int(numRead) != numEntries {
-		panic(fmt.Sprintf("Expected %d cache entries but received %d", numEntries, numRead))
-	}
+		body, err := ioutil.ReadAll(rsp.Body)
+		tutils.ExpectNoError(err)
+		numRead, err := strconv.ParseInt(string(body), 10, 64)
+		tutils.ExpectNoError(err)
+		if int(numRead) != numEntries {
+			panic(fmt.Sprintf("Expected %d cache entries but received %d", numEntries, numRead))
+		}
```

Comment:
Unnecessary formatting changes

Reply:
Yes, @ryanemerson but this was adding by IDE. Will revert.